frontend/src/pages/ActivityReport/index.js

```diff
@@ -178,36 +178,6 @@ export const unflattenResourcesUsed = (array) => {
   return array.map((value) => ({ value }));
 };
 
-/**
```

Comment:
removed as we haven't used this since the switchover, that way we don't have to test this